File: src/main/java/li/cil/occ/mods/mekanism/ModMekanism.java
package li.cil.occ.mods.mekanism;

import li.cil.oc.api.Driver;
import li.cil.occ.mods.IMod;
import net.minecraft.item.ItemStack;

import java.util.Map;

public final class ModMekanism implements IMod {
    @Override
    public String getModId() {
        return "Mekanism";
    }

    @Override
    public void initialize() {
        Driver.add(new DriverBasicMachine());
        Driver.add(new DriverDigitalMiner());
        Driver.add(new DriverElectrolyticSeperator());
        Driver.add(new DriverEnergyCube());
        Driver.add(new DriverFactory());
        Driver.add(new DriverGenerator());
        Driver.add(new DriverMetallurgicInfuser());
        Driver.add(new DriverTeleporter());

    }

    @Override
    public void populate(final Map<String, Object> map, final ItemStack stack) {
    }
}

File: src/main/scala/li/cil/oc/integration/mekanism/ModMekanism.scala
package li.cil.oc.integration.mekanism

import li.cil.oc.api
import li.cil.oc.integration.ModProxy
import li.cil.oc.integration.Mods

object ModMekanism extends ModProxy {
  override def getMod = Mods.Mekanism

  override def initialize(): Unit = {
    api.Driver.add(ConverterGasStack)
  }
}
